python/tvm/schedule.py

```diff
@@ -45,6 +45,9 @@ def access_ptr(self, access_mask, ptr_type="handle", content_lanes=1):
             The number of lanes for the data type. This value
             is greater than one for vector types.
 
+        offset: int, optional
+		    The offset of pointer.
```

Comment:
specify what is the unit here(number of elemenet?)

Reply:
Refined